pythonExecSource/src/main/python/pyExecConnector.py

```diff
@@ -562,13 +583,43 @@ async def run_python_code(self, ctx: dict, code_text: str, script_to_fetch: str,
                                      VantiqConnector.ERROR_PARAMETERS: []}
                 if error_msg:
                     await self.connection.send_query_error(ctx, error_msg)
+                    return
```

Comment:
restructuring to better report exceptions requires a couple of explicit returns.